### PR TITLE
(maint) remove git & package methods from acceptance

### PR DESCRIPTION
**Pull request by er0ck**
This change removes unused install methods from acceptance

[skip ci]

### CODE REVIEW

**Comment by MikaelSmith** [comment]
:+1:

File: acceptance/Rakefile [hunk 1]
```diff
@@ -197,82 +196,58 @@ May specify project SHA environment variable, which can be a branch, release ver
...
-    preserved = list_preserved_configurations
-    print_preserved(preserved)
-    config_number = (ENV['CONFIG_NUMBER'] || 0).to_i
-    config_path = preserved[config_number][0]
-    puts "Using ##{config_number}: #{config_path}"
-    beaker_test(beaker_run_type,
-      :hosts_file => "log/latest/#{PRESERVED_HOSTS_FILENAME}"
-    )
+    EOS
+    task :test_against_preserved_hosts do
```
Comment (MikaelSmith) [inline acceptance/Rakefile]: This is now part of the ci:test namespace?

Comment (er0ck) [inline acceptance/Rakefile]: woops.  removed an extra end and didn't catch the namespacing...  will fix

**Comment by er0ck** [comment]
@MikaelSmith fixed...

File: acceptance/Rakefile (hunk 1 above)
Comment (MikaelSmith) [inline acceptance/Rakefile]: I think know they're no longer in any namespace. They were originally in the `ci` namespace. I'd expect to see a lot less line changes for this PR, indentation of tasks shouldn't really change.

**Comment by er0ck** [comment]
this should probably just wait for the changes coming via the POC against puppet to unify the puppet agent component acceptance workflows.  these should also be against stable.

**Pull request closed without merging**